jwst/master_background/master_background_nrs_slits_step.py

```diff
@@ -254,7 +254,7 @@ def _calc_master_background(self, data, user_background=None):
             mb_multislit = nirspec_utils.map_to_science_slits(pre_calibrated, master_background)
 
             # Now that the master background is pretending to be science,
-            # walk backwards through the steps to uncalibrate, using the
+            # walk backwards through the steps to decalibrate, using the
```

Comment:
This code region has `decalibrate`, `uncalibrate`, and `pre-calibrated`. The first and last of these are in dictionaries. I can understand `uncalibrated` (which is in the dictionary). But I personally had trouble with the idea of actively uncalibrating (not in the dictionary).

Happy to drop. But I'd also like to learn about these.

Reply:
We should stick with the original "uncalibrate" here.